### PR TITLE
feat(ticket#30): Implementation of the Missing Shoes List and Header

### PR DESCRIPTION
**Pull request by nadirhussain786**
## Summary 

This PR introduces the UI for the "Missing Shoes" list and the Header. Additionally, it addresses the mobile responsiveness of the sidebar. The changes include:

- A table displaying the list of reported missing shoes.
- A header featuring breadcrumbs and an `Add Missing Shoes` button.
- The `Add Missing Shoes` button is displayed only when the current pathname is `/missing-shoes`.
- A fix for the sidebar, ensuring it is displayed correctly on mobile devices.

### Resources

image

image

image

### CODE REVIEW

File: src/app/(app)/missing-shoes/components/MissingShoes.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  TableRow,
+} from "@/components/ui/table";
+import { MissingShoeStatus, Shift } from "@/constant/constant";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { Plus } from "lucide-react";
```
Comment (muhammad-bilal-ali-khokhar) [inline src/app/(app)/missing-shoes/components/MissingShoes.tsx:18]: `plusIcon`

File: src/app/(app)/missing-shoes/components/MissingShoes.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        cell: ({ row }) => <div>{row.getValue("date")}</div>,
+      },
+      {
+        accessorKey: "shift",
+        header: "shift",
+        cell: ({ row }) => <div>{row.getValue("shift")}</div>,
+      },
+      {
+        accessorKey: "size",
+        header: "size",
```
Comment (ghulamhafeez) [inline src/app/(app)/missing-shoes/components/MissingShoes.tsx:89]: header size, color & shift not in capital case

File: src/app/(app)/missing-shoes/components/MissingShoes.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+} from "@/components/ui/table";
+import { MissingShoeStatus, Shift } from "@/constant/constant";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { Plus } from "lucide-react";
+import React, { useMemo } from "react";
```
[the remark this answers is not in the document]
Comment (nadirhussain786) [reply src/app/(app)/missing-shoes/components/MissingShoes.tsx]: Fixed

File: src/app/(app)/missing-shoes/components/MissingShoes.tsx (hunk 2 above)
Comment (nadirhussain786) [reply src/app/(app)/missing-shoes/components/MissingShoes.tsx]: Fixed

File: src/app/(app)/missing-shoes/components/MissingShoes.tsx (hunk 1 above)
Comment (nadirhussain786) [reply src/app/(app)/missing-shoes/components/MissingShoes.tsx]: Fixed

**Review by Mubashar1009 — APPROVED** [review]

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

**Pull request merged**